app/src/main/java/fr/free/nrw/commons/upload/FileUtils.java

```diff
@@ -43,7 +43,7 @@
     /**
      * Get SHA1 of file from input stream
      */
-    static String getSHA1(InputStream is) {
+    public static String getSHA1(InputStream is) {
```

Comment:
Usually its  not a good practice to change the access modifiers of a method just for writing tests for it.